com.milesalan.mepo.yml

```
@@ -0,0 +1,110 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.ziglang"]
+extension-tag: '21.08'
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-documents
+
+build-options: { append-path: "/usr/lib/sdk/ziglang" }
```

```suggestion
build-options:
    append-path: /usr/lib/sdk/ziglang/bin
```

this is YAML. Also I think you wanted the `bin` directory.

---

Fixed